bin/middlewares/map_style.js

```diff
@@ -0,0 +1,40 @@
+const styleBuilder = require('@qwant/map-style-builder')
+const styleConfigure = require('mapbox_style_configure')
+const qwantStyle = require('@qwant/qwant-basic-gl-style/style.json')
+const path = require('path')
+
+
+module.exports = function (config, languages) {
+
+  /* pre-build style on server start */
+  let options = {
+    output: 'production', // 'debug' | 'production' | 'omt'
+    outPath : __dirname + '/../public/mapstyle',
+    i18n : true,
+    icons : true,
+    pixelRatios : [1,2],
+    styleDir : path.resolve(path.join(__dirname, '..', '..', 'node_modules', '@qwant' , 'qwant-basic-gl-style')),
+    conf : {tileserver_base : '{tileserver_base}', tileserver_poi : '{tileserver_poi}', spriteserver : '{spriteserver}', fontserver : '{fontserver}'},
+    ignoreSprite : true
```

Comment:
Because of this flag, `sprite` property will be removed from the style object.  
So the spriteserver is not present in the result :(

Reply:
i think,i'll keep the old boolean (sprite exist) for transform call & use build sprite for the spritezero call.

Reply:
cool ! please create a PR on map-style-builder

Reply:
https://github.com/QwantResearch/map-style-builder/pull/23